### PR TITLE
Get Rack Locations from Sunbird

### PR DESCRIPTION
**Pull request by adhil0**
- Add ability to add Data Centers, Data Center rooms, Racks to GLPI via Sunbird
- Update Sunbird config to be more customizable

There is some repeated code between the new functions, so the next steps after this PR is to see if we can reduce the amount of repeated code.

### CODE REVIEW

File: population/create_glpi_computer_redfish.py [hunk 1]
```diff
@@ -156,6 +179,12 @@ def main() -> None:
     sku = args.sku
     switch_config = args.switch_config
     no_verify = args.no_verify
+    sunbird_username = args.sunbird_username
+    sunbird_password = args.sunbird_password
+    if args.sunbird_url:
+        sunbird_url = validate_url(args.sunbird_url)
+    else:
+        sunbird_url = args.sunbird_url
```
Comment (dkosteck) [inline population/create_glpi_computer_redfish.py:187]: So you're just setting this to None if there is no sunbird args passed? Maybe setting to None would be more explicit?

File: population/create_glpi_computer_redfish.py [hunk 2]
```diff
@@ -619,6 +660,31 @@ def post_to_glpi(  # noqa: C901
         print(str(computer_response) + "\n")
         COMPUTER_ID = computer_response.json()["id"]
 
+    # Get Rack Location from Sunbird if relevant flags are provided
+    if all(
```
Comment (dkosteck) [inline population/create_glpi_computer_redfish.py:664]: So this is just to short circuit all sunbird logic if one of the args is not present, correct?

**Review by dkosteck — APPROVED** [review]
Looks good, just a couple of minor (optional) comments

File: population/create_glpi_computer_redfish.py (hunk 2 above)
Comment (adhil0) [reply population/create_glpi_computer_redfish.py:664]: Correct - I modified the comment slightly to be more clear.

File: population/create_glpi_computer_redfish.py (hunk 1 above)
Comment (adhil0) [reply population/create_glpi_computer_redfish.py]: Good idea - addressed in latest commit

**Pull request merged**